### DISCUSSION TITLE
Spotify Stats

### DISCUSSION THREAD

**Discussion by edwin100x** [skymethod/op3#36] [Q&A] [answered]
Hi there, I couldn't find the answer anywhere. Do we get Spotify stats when uploading a video to Spotify?

**Comment by johnspurlock-skymethod** [comment, marked as answer]
No, when you upload a video version, Spotify will use their own copy - even when playing only audio!  This means these Spotify plays won't appear in your host stats or any 3p prefix stats like OP3.

https://podnews.net/update/spotify-video-all-podcasts
<img width="881" alt="image" src="https://github.com/user-attachments/assets/231d9c02-b02e-4c91-924e-65d261a72fcb">

Hope that helps!
